src/halomod/functional.py

```diff
@@ -1,5 +1,5 @@
 r"""
-Module which defines a :func:`get_halomodel` function.
+Module defining a :func:`get_halomodel` function.
```

Comment:
Make this a bit more general -- we may add other functions to this module! Something like "Module defining functional approaches to generating halo model quantities."